src/relay/Command.ts

```diff
@@ -48,6 +48,33 @@ export class Command {
         );
     };
 
+    static createEVMContractCallWithTokenCommand = (commandId: string, relayData: RelayData, args: CallContractWithTokenArgs) => {
```

Comment:
Where is this used?

Reply:
https://github.com/axelarnetwork/axelar-local-dev/blob/b34e8277fc984d5856b2755e196fa90db4f95d84/src/relay/EvmRelayer.ts#L367